DateRangeBox: add code snippets and extended description #5043 (#5111)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDateRangeBox/1 Configuration/endDateText.md

---
---
##### shortDescription

> The read-only property that holds the text displayed by the end date input field. 

Reviewer: Can we just say "returns" instead of "read-only property that holds"? That is our usual word for such cases and it implies that the property is "read-only". 

```suggestion
Returns the text displayed by the end date input field. 
```
